src/client/mod.rs

```diff
@@ -287,6 +292,58 @@ impl<'a> ClientBuilder<'a> {
 
         self
     }
+
+    /// Sets how long the http client should wait for the entire response from discord API.
```

Comment:
Discord is a proper noun in this context, thus it should be capitalised.
```suggestion
    /// Sets how long the http client should wait for the entire response from the Discord API.
```